.github/workflows/continuous-integration.yml

```diff
@@ -102,19 +66,26 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+        continue-on-error: ${{ matrix.experimental }}
```

Comment:
Sorry I did not catch this one. This is meant to be on the `job` level, not the `step` level, i.e. if the job fails, it does not block later jobs that depend on it, compared to if on step, it does not block later steps

Reply:
I see! That's indeed better. I have included that in the new PR #1090